ionoscloud/resource_vcpu_server.go

```diff
@@ -27,6 +27,12 @@ func resourceVCPUServer() *schema.Resource {
 				Required:         true,
 				ValidateDiagFunc: validation.ToDiagFunc(validation.StringIsNotWhiteSpace),
 			},
+			"hostname": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				Description:      "The hostname of the resource. Allowed characters are a-z, 0-9 and - (minus). Hostname should not start with minus and should not be longer than 63 characters.",
+				ValidateDiagFunc: validation.ToDiagFunc(validation.All(validation.StringIsNotWhiteSpace, validation.StringLenBetween(1, 63))),
```

Comment:
@cristiGuranIonos same comment as the one from above related to validation.

Reply:
will keep it, it doesn't hurt for this field as it's optional